modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.1.10.adoc

```diff
@@ -0,0 +1,24 @@
+[#maint-3-1-10]
+== 3.1.10 -- November 2024
+
+Version 3.1.10 for {param-title} delivers the following features and enhancements:
+
+=== Enhancements
+
+None for this release
+
+=== Issues and Resolutions
+// Lite Core Start
+* https://issues.couchbase.com/browse/CBL-6282[CBL-6282 -- Replicator starts slowly for big databases]
+
+* https://issues.couchbase.com/browse/CBL-6377[CBL-6377 -- Fixed crash when calling onWebSocketGotTLSCertificate callback after the connection is closed]
+// Lite Core End
```

Comment:
Aha, I see, yes makes more sense for this one that has additional non-LiteCore tickets 👍